src/rates.ts

```diff
@@ -166,6 +166,7 @@ export const getExchangeRates = async (
     )
     return out
   } catch (e) {
+    console.log(e)
```

Comment:
Use logger and add additional context to the log so we can see where it's coming from